PURL-TYPES.rst

```diff
@@ -578,6 +578,25 @@ swift
       pkg:swift/github.com/Alamofire/Alamofire@5.4.3
       pkg:swift/github.com/RxSwiftCommunity/RxFlow@2.12.4
 
+
+nix
+---
+
+``nix`` for Nix derivations:
+
+- The default package repository is <https://github.com/NixOS/nixpkgs> but the ``repository`` qualifier can override the package repository.
```

Comment:
Does this field even matter at all? With just the PURL we cannot reproduce the drv path anyways, there's a lot of other details (like rev, system, ...) we'd need. Also, the line gets blurry if you have a repo that imports nixpkgs, but overrides an expression, what should this be set to?

Inclined to remove this, this can only be a hint at most.

Reply:
With this field being mostly informative I feel we'd want a `nar` package type or at least a `nar` field to better identify where you got the package from if using a cache like `cache.nixos.org`. ommitted or something for if you built it locally?

Reply:
It is not identifying the package. If you have a nar sha256 hash, you can fetch it from any content-addressed store.

Reply:
"what" being covered by the hash, but "where" can also useful for some governance.
Also if you can find that hash in your content-addressed store then great, but if you can't you might at least want a possible alternative location to check to find the same artifact described by the PURL.